src/translations/fr.json

```diff
@@ -17,6 +17,7 @@
 
     "moveStudyNotification": "L'étude a été déplacée de {oldStudyPath} à {studyPath}",
 
+    "invalidateCopyPastOfNode": "Invalidation du collage du noeud suite à la modification du neoud copié",
```

Comment:
invalidateCopy~~Past~~OfNode
I propose this text :
"Invalidation de la copie du nœud suite à modification"

Reply:
It's fixed